addons/point_of_sale/static/src/app/utils/use_idle_timer.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "@odoo/owl";
+
+const UserPresenceEvents = [
+    "mousemove",
+    "mousedown",
+    "touchstart",
+    "touchend",
+    "touchmove",
+    "click",
+    "scroll",
+    "keypress",
+];
+
+export function useIdleTimer(steps, onAlive) {
+    const state = useState({
+        timeout: new Set(steps.map((s) => s.timeout)),
+        idle: false,
+        time: 0,
+    });
+
+    const checkSteps = () => {
+        for (const step of steps) {
+            if (step.timeout === state.time * 1000) {
+                state.idle = true;
+                step.action();
+            }
+        }
+    };
+
+    const onMove = () => {
+        if (state.idle) {
+            state.idle = false;
+            onAlive();
+        }
+        state.time = 0;
+    };
+
+    for (const event of UserPresenceEvents) {
+        window.addEventListener(event, onMove);
```

Comment:
```suggestion
        useExternalListener(window, event, onMove);
```

We should make sure that when the component is unmounted, we also remove the listeners.

Reply:
I changed it, but Chrome is never unmounted.